conan/internal/api/detect_api.py

```diff
@@ -5,6 +5,7 @@
 import textwrap
 
 from conan.api.output import ConanOutput
+from conans.errors import ConanException
```

Comment:
New imports should be 
```suggestion
from conan.errors import ConanException
```

Reply:
We need to check with @czoido, this was done automatically by the IDE, we need to teach pycharm not to screw up